Marlin/src/HAL/STM32/timers.cpp

```diff
@@ -29,26 +29,41 @@
 
 #define NUM_HARDWARE_TIMERS 2
 
+// Default timer priorities. Override by specifying alternate priorities in the board pins file.
+// The TONE timer is not present here, as it currently cannot be set programmatically. It is set
+// by defining TIM_IRQ_PRIO in the variant.h or platformio.ini file, which adjusts the default
+// priority for STM32 HardwareTimer objects.
+#define SERVO_TIMER_IRQ_PRIO_DEFAULT     1 // Requires tight bit timing to communicate reliably with TMC drivers
+#define SWSERIAL_TIMER_IRQ_PRIO_DEFAULT  1 // Requires tight PWM timing to control a BLTouch reliably
+#define STEP_TIMER_IRQ_PRIO_DEFAULT      2
```

Comment:
Swap comments or defines.

Reply:
I had to stare at it a second, but I see the problem now. Thanks!